<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogField</class>
 <widget class="QDialog" name="dialogField">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="Shadow" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>605</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#Shadow{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 			rgba(85, 98, 112, 255), stop:1 rgba(255, 107, 107, 255));
color: rgb(220, 220, 220);
}</string>
   </property>
   <widget class="QLabel" name="swords">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>-50</y>
      <width>331</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>86</pointsize>
     </font>
    </property>
    <property name="text">
     <string>âš”</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="editArea">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>721</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border-radius:10px;
background-color: rgb(238, 238, 236);
padding:  20px;
padding-left: 20px;</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>530</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#login{
	background-color: rgba(85, 98, 112, 255);
	color: rgba(255, 255, 255, 200);
	border-radius:5px;
}
QPushButton#login:pressed{
	padding-left:5px;
	padding-top: 5px;
	background-color: rgba(255, 107, 107, 255);
	background-position:calc(100% - 10px)center;
}
QPushButton#login:hover{
	background-color: rgba(255, 107, 107, 255);
}</string>
    </property>
    <property name="text">
     <string>Start Bot</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="sendEmails">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>721</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border-radius:10px;
background-color: rgb(238, 238, 236);
padding:  20px;
padding-left: 20px;</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Emails or Email the Generated .txt file will be sent to 
                    separate the emails with a comma</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add faceBook Groups links here 
add link and press enter to add next(No commas  (,,,,,,,,,,))</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
